src/js/components/event/index.js

```diff
@@ -39,47 +37,37 @@ class Event extends React.Component {
     if (this.props.userIsHost && this.props.isPoll) {
       // changed from this.props.params.event_id
       return (
-        <View>
-          <HostPoll
-            tally={ this.props.tally }
-            event={ this.props.event }
-            event_id={ this.props.event.event_id }
-            finalChoices={ this.props.finalChoices }
-            hostEventChoices={ this.props.hostEventChoices }
-            handleConfirmEvent={ this.props.handleConfirmEvent }
-          />
-        </View>
+        <HostPoll
+          tally={ this.props.tally }
+          event={ this.props.event }
+          event_id={ this.props.event.event_id }
+          finalChoices={ this.props.finalChoices }
+          hostEventChoices={ this.props.hostEventChoices }
+          handleConfirmEvent={ this.props.handleConfirmEvent }
+        />
       );
-    } else if (!this.props.userIsHost && this.props.isPoll) {
+    } else if (true) {
```

Comment:
@minaorangina  Here I placed true to just render the invitee screen, we can change that to the way it was